src/main/java/uk/gov/hmcts/reform/sscs/service/EvidenceManagementService.java

```diff
@@ -32,11 +34,21 @@ public UploadResponse upload(List<MultipartFile> files) {
         String serviceAuthorization = authTokenGenerator.generate();
 
         try {
-            return documentUploadClientApi
-                    .upload(DUMMY_OAUTH_2_TOKEN, serviceAuthorization, files);
+            return documentUploadClientApi.upload(OAUTH_2_TOKEN, serviceAuthorization, files);
         } catch (HttpClientErrorException httpClientErrorException) {
+            log.error("Doc Store service failed to upload documents...");
```

Comment:
Log the actual  exception httpClientErrorException.

Reply:
done